docs/sqlalchemy.rst

```diff
@@ -284,6 +284,58 @@ The resulting object will look like this::
     change, the `UPDATE`_ statement sent to CrateDB will include all of the
     ``ObjectArray`` data.
 
+.. _geopoint:
+.. _geoshape:
+
+``Geopoint`` and ``Geoshape``
+.............................
+
+The CrateDB SQLAlchemy dialect provides two geospatial types:
+- ``Geopoint``, which represents a longitude and latitude coordinate
+- ``Geoshape``, which is used to store geometric `GeoJSON geometry objects`_.
+
+To use these types, you can create columns using them::
+
+    >>> class City(Base):
+    ...
+    ...    __tablename__ = 'cities'
+    ...    name = sa.Column(sa.String, primary_key=True)
+    ...    coordinate = sa.Column(types.Geopoint)
+    ...    area = sa.Column(types.Geoshape)
+
+To create a geopoint, you can either define it as a tuple of
+``(longitude, latitude)``::
+
+    >>> point = (139.76 35.68)
+
+or as a geojson ``Point`` object::
```

Comment:
it's generally not considered good style to split a sentence up with an example. can you reword so each leading sentence is complete?